sklearn/ensemble/voting_classifier.py

```diff
@@ -54,6 +54,9 @@ class VotingClassifier(_BaseComposition, ClassifierMixin, TransformerMixin):
         the sums of the predicted probabilities, which is recommended for
         an ensemble of well-calibrated classifiers.
 
+    verbose : bool, optional (default=False)
```

Comment:
why not use an int here? I find it weird to remove a level of control here.

Reply:
@jnothman was suggesting in previous comments (see [1](https://github.com/scikit-learn/scikit-learn/pull/10974#discussion_r181550238) and [2](https://github.com/scikit-learn/scikit-learn/pull/10974#issuecomment-381438660)) that `VotingClassifier` isn't used for a huge number of estimators. And I think that the int level of control is useless if the number of estimators is smaller than, let's say, 50. Do you think that it is used for large amounts of classifiers? And that we should keep it as int?

Reply:
yes. the estimator itself can have different verbosity levels